RetroBar/Controls/TaskButton.xaml.cs

```diff
@@ -180,6 +181,33 @@ private void CloseMenuItem_OnClick(object sender, RoutedEventArgs e)
             Window?.Close();
         }
 
+        private void EndTaskMenuItem_OnClick(object sender, RoutedEventArgs e)
+        {
+            if (Window != null)
+            {
+                ForceEndTask();
+            }
+        }
+
+        private void ForceEndTask()
+        {
+            try
+            {
+                uint processId;
+                NativeMethods.GetWindowThreadProcessId(Window.Handle, out processId);
```

Comment:
We could use `Window.ProcId` instead (unless there is a reason for not doing so that I'm missing!)